src/Components/Message/Banner.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+	<div class="c-application c-banner" :class="[computedType]">
+		<div class="c-banner--img">
+			<slot name="img" />
+		</div>
+		<div class="c-banner--container">
+			<div class="c-banner--container--title">
+				<slot name="title" />
+			</div>
+			<div class="c-banner--container--discription mt-8">
```

Comment:
```suggestion
			<div class="c-banner--container--description">
```
description 오타가 있네요ㅠㅠ storybook에도 수정해야합니다.

`mt-8` 추가 클래스보다는 `c-banner--container-description` 클래스에게 style을 주는 것이 어떨까요? 차후에 <style>부분만 보면 되니까 관리하기에 용이할 것 같아요.